### DISCUSSION TITLE
我用这玩意为啥老是更换ip啊

### DISCUSSION THREAD

**Discussion by peinuonuo** [shadowsocks/shadowsocks-windows#3181] [Q&A]
我买的是日本的服务器 但是不知道咋回事 用了没几分钟就自动给ip换到香港区了 然后过一会再换回日本 这是个什么玩意

**Comment by peinuonuo** [comment]
![Uploading 1621222190(1).jpg…]()
![Uploading 1621222205(1).jpg…]()

**Comment by chenshaoju** [comment]
不要选择负载均衡或高可用模式，直接选择你要用的服务器即可。

**Reply by peinuonuo** [reply]
大佬有搭建vpn的详细教程么 我给负载均衡勾掉之后好像访问不了了

**Reply by chenshaoju** [reply]
这说明你选择的服务器是坏的，换一个把。

**Reply by peinuonuo** [reply]
搞好了搞好了 重新搞了一下

**Reply by peinuonuo** [reply]
不知道咋回事小飞机老断开 服务器没啥问题 就是难搞
